src/DssSpell.t.sol

```diff
@@ -614,369 +604,424 @@ contract DssSpellTest is DssSpellTestBase {
-        assertEq(dai.balanceOf(address(rwaurn_014)), drawAmount - pushAmount, "RWA014: Dai not sent to destination after push()");
+        assertEq(dai.balanceOf(address(rwa015AOutputConduit)), 0, "RWA015-A: Output conduit still holds Dai after quit()");
+        assertEq(psmGem.balanceOf(address(this)), pushAmount / daiPsmGemDiffDecimals, "RWA015-A: Psm GEM not sent to destination after push()");
+        assertEq(dai.balanceOf(address(rwa015AUrn)), drawAmount - pushAmount, "RWA015-A: Dai not sent to destination after push()");
 
         // as we have SF 0 we need to pay exectly the same amount of DAI we have drawn
         uint256 daiToPay = drawAmount;
 
         // Note: In the version of inputConduit for this deal `push` is permissionles
         // // wards
-        // GodMode.setWard(address(rwaconduitinurn_014), address(this), 1);
+        // GodMode.setWard(address(rwa015AInputConduitUrn), address(this), 1);
         // // may
-        // rwaconduitinurn_014.mate(address(this));
-        // assertEq(rwaconduitinurn_014.may(address(this)), 1);
+        // rwa015AInputConduitUrn.mate(address(this));
+        // assertEq(rwa015AInputConduitUrn.may(address(this)), 1);
 
         // transfer PSM GEM to input conduit
-        psmGem.transfer(address(rwaconduitinurn_014), pushAmount / daiPsmGemDiffDecimals);
-        assertEq(psmGem.balanceOf(address(rwaconduitinurn_014)), pushAmount / daiPsmGemDiffDecimals, "RWA014: Psm GEM not sent to input conduit");
-        
+        psmGem.transfer(address(rwa015AInputConduitUrn), pushAmount / daiPsmGemDiffDecimals);
+        assertEq(psmGem.balanceOf(address(rwa015AInputConduitUrn)), pushAmount / daiPsmGemDiffDecimals, "RWA015-A: Psm GEM not sent to input conduit");
+
         // input conduit 'push()' to the urn
-        rwaconduitinurn_014.push();
+        rwa015AInputConduitUrn.push();
 
-        assertEq(dai.balanceOf(address(rwaurn_014)), daiToPay, "Balance of the URN doesnt match");
+        assertEq(dai.balanceOf(address(rwa015AUrn)), daiToPay, "Balance of the URN doesnt match");
 
         // repay debt and free our collateral
-        rwaurn_014.wipe(daiToPay);
-        rwaurn_014.free(1 * WAD);
+        rwa015AUrn.wipe(daiToPay);
+        rwa015AUrn.free(1 * WAD);
 
-        // check if we get back RWA014 Tokens
-        assertEq(rwagem_014.balanceOf(address(this)), 1 * WAD, "RWA014: gem not sent back to the caller");
+        // check if we get back RWA015 Tokens
+        assertEq(rwa015AGem.balanceOf(address(this)), 1 * WAD, "RWA015-A: gem not sent back to the caller");
 
         // check if we have 0 collateral and outstanding debt in the VAT
-        (ink, art) = vat.urns("RWA014-A", address(rwaurn_014));
-        assertEq(ink, 0, "RWA014: bad `ink` after free()");
-        assertEq(art, 0, "RWA014: bad `art` after wipe()");
+        (ink, art) = vat.urns("RWA015-A", address(rwa015AUrn));
+        assertEq(ink, 0, "RWA015-A: bad `ink` after free()");
```

Comment:
I don't mind the changes to RWA015 -> RWA015-A in tests and similar changes in varnames as these are non-functional, just a note for future reference that it will make it harder to search the archives.

Reply:
It's easier to search for the specific test than for the specific variable. `grep '.*_OPERATOR_LOCK_DRAW_WIPE_FREE' archive/**'